common/uint160.go

```diff
@@ -94,6 +94,11 @@ func (f Uint160) MarshalJSON() ([]byte, error) {
 	return []byte("\"" + str + "\""), err
 }
 
+func (f Uint160) UnmarshalJSON(in []byte) (err error) {
+	f, err = ToScriptHash(string(in[1 : len(in)-1]))
```

Comment:
Need to validate `len(in) > 1` to prevent panic

Reply:
silly me. fixed.